pages/index.js

```diff
@@ -55,14 +55,18 @@ export default function Home() {
                 setActiveTab(key);
                 return;
             }
-            const wasm_bytes_response = lfortran_funcs.compile_code(sourceCode);
+            const wasm_bytes_response = lfortran_funcs.compile_code(sourceCode);            
             if (wasm_bytes_response) {
+                const start_compile = performance.now();
```

Comment:
We need to measure the compilation time for `lfortran_funcs.compile_code(sourceCode)`. I think it would be as follows:
```suggestion
            const start_compile = performance.now();
            const wasm_bytes_response = lfortran_funcs.compile_code(sourceCode);
            const end_compile = performance.now();
            if (wasm_bytes_response) {
```